src/packages/engine-core/src/NodeEngine.ts

```diff
@@ -819,12 +832,12 @@ ${this.lastErrorLog.fields.file}:${this.lastErrorLog.fields.line}:${this.lastErr
 
         this.url = `http://localhost:${this.port}`
 
-        // don't wait for this
-        ;(async () => {
-          const engineVersion = await this.version()
-          debug(`Client Version ${this.clientVersion}`)
-          debug(`Engine Version ${engineVersion}`)
-        })()
+          // don't wait for this
+          ; (async () => {
```

Comment:
What is propose of the `;` here?

Reply:
It's prettier https://github.com/prettier/prettier/issues/7423

Reply:
Ahh ok, thanks